opentelemetry-sdk/tests/metrics/test_metric_cardinality_limit.py

```diff
@@ -0,0 +1,40 @@
+
+from logging import WARNING
+from unittest import TestCase
+
+from opentelemetry.sdk.metrics import MeterProvider
+
+from opentelemetry.sdk.metrics.export import (
+    InMemoryMetricExporter,
+    PeriodicExportingMetricReader,
+)
+
+class TestMetricCardinalityLimit(TestCase):
+
+    def setUp(self):
+        self.exporter = InMemoryMetricExporter()
+        self.reader = PeriodicExportingMetricReader(exporter=self.exporter, export_interval_millis=500)
+        self.meter_provider = MeterProvider(metric_readers=[self.reader])
+        self.meter = self.meter_provider.get_meter("test_meter")
+
+    def test_metric_cardinality_limit(self):
```

Comment:
If we are introducing the ability to configure the value we should have a test for that too.